src/seer/seer.py

```diff
@@ -68,14 +68,11 @@ def breakpoint_trends_endpoint():
         data = request.get_json()
         txns_data = data["data"]
 
-        # new format has zerofilled parameter - if it's not being sent to microservice default value is True
-        zerofilled = data.get("zerofilled", True)
-
         sort_function = data.get("sort", "")
-
         allow_midpoint  = data.get("allow_midpoint", "1") == "1"
 
-        lower_limit_trend_percentage = float(data.get('trend_percentage()', 0.1))
+        min_pct_change = float(data.get('trend_percentage()', 0.1))
+        min_ms_change = float(data.get('min_ms_change()', 25.0))
```

Comment:
I think we should default this to 0 and let the caller decide. Functions use nanoseconds as the unit so forcing a unit here is not ideal.

```suggestion
        min_change = float(data.get('min_change()', 0))
```